pkg/reconciler/pubsub/resources/topic_test.go

```diff
@@ -134,6 +155,7 @@ func TestMakeTopicWithCloudSchedulerSource(t *testing.T) {
 			Name:      "scheduler-name",
 			Labels: map[string]string{
 				"receive-adapter": "storage.events.cloud.google.com",
```

Comment:
seems an old copy/paste.. should be `scheduler.events.cloud.google.com`?

Reply:
done